db/migrate/20230328034956_add_sidebar_section_id_index_to_sidebar_section_link.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class AddSidebarSectionIdIndexToSidebarSectionLink < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def up
+    execute <<~SQL
+    DROP INDEX CONCURRENTLY IF EXISTS idx_sidebar_section_links_on_sidebar_section_id
+    SQL
+
+    execute <<~SQL
+    CREATE UNIQUE INDEX CONCURRENTLY idx_sidebar_section_links_on_sidebar_section_id
+    ON sidebar_section_links (sidebar_section_id, user_id, position)
+    SQL
+
+    execute <<~SQL
+    DROP INDEX CONCURRENTLY IF EXISTS links_user_id_section_id_position
+    SQL
+  end
+
+  def down
+    execute <<~SQL
+    DROP INDEX CONCURRENTLY IF EXISTS idx_sidebar_section_links_on_sidebar_section_id
+    SQL
+
+    add_index :sidebar_section_links,
```

Comment:
Shouldn't we be adding this one concurrently as well?

Reply:
Does not really matter ... its in the downwards train so if it fails it is re-runnable and it was created that way in the first place anyway